static/components.ts

```diff
@@ -980,3 +938,275 @@ export function getDeviceViewWith(
         },
     };
 }
+
+// =============================================================================
```

Comment:
Should this comment be here?

Reply:
No, I'll tidy that up before we merge. It really wanted to try and make a big fanfare about this :)